.buildkite/react-native-cli-pipeline.yml

```diff
@@ -282,7 +282,7 @@ steps:
     env:
       DEBUG: true
       LANG: "en_US.UTF-8"
-      DEVELOPER_DIR: "/Applications/Xcode11.7.app"
+      DEVELOPER_DIR: "/Applications/Xcode11.app"
```

Comment:
It's probably not a problem if the build works, but I notice that RN 0.63 builds with Xcode 12 in the RN CLI pipeline.

Reply:
Hmm the only reason that this would have been added is because that it was building on the old ms-mac-10-15 queue. I'll double check.

Reply:
This step used to be on the `opensource-mac-rn` queue which was the old 10-15 macs

Reply:
Yeah, there's a fair bit of history in these pipelines.  If they both build I think it's fine.